twisted/protocols/memcache.py

```diff
@@ -299,20 +305,22 @@ def cmd_ERROR(self):
         cmd.fail(NoSuchCommand())
 
 
-    def cmd_CLIENT_ERROR(self, errText):
+    def cmd_CLIENT_ERROR(self, errBytes):
```

Comment:
I believe this breaks our backwards compatibility policy, because it's renaming parts of our public API :(

It does make it clearer, though, so I believe that maybe something like:

```
def somefunc(self, errBytes=None, errText=None):
    if errText:
        raise DeprecationWarning("Passing 'errText' as a kwarg to somefunc is deprecated since Twisted 16.4.0")
    if errText and errBytes is None:
        errBytes = errText
    elif errText and errBytes:
        raise ValueError("Please specify only only one argument (errBytes).")
```

or something...

cc @rodrigc , @glyph